app/models/mappers/check-activities.js

```diff
@@ -1,6 +1,164 @@
 const { formatToGds } = require('../../lib/date-helpers')
 const { cleanUserDisplayName } = require('../../lib/model-helpers')
 
+/**
```

Comment:
This section can be helpful but it's a lot of information. Do you think it's worth keeping in